docs/reference/ccr/index.asciidoc

```diff
@@ -2,28 +2,172 @@
+its configured limits, no additional read requests are sent. This configuration
+provides a back-pressure against read requests, allowing the follower shard
+to resume sending read requests when the write buffer is no longer full.
+
+To manage how operations are replicated from the leader index, you can
+configure settings when
+<<ccr-getting-started-follower-index,creating the follower index>>.
+
+The follower index automatically retrieves some updates applied to the leader
+index, while other updates are retrieved as needed:
+
+[cols="3"]
+|===
+h| Update type h| Automatic  h| As needed
+| Alias        | {yes-icon} | {no-icon}
+| Mapping      | {no-icon}  | {yes-icon}
+| Settings     | {no-icon}  | {yes-icon}
+|===
+
+For example, changing the number of replicas on the leader index is not
+replicated by the follower index, so that setting might not be retrieved.
+
+NOTE: You cannot manually modify the mapping or alias of a follower index.
+
+If you apply a non-dynamic settings change to the leader index that is
+needed by the follower index, the follower index closes itself, applies the
+settings update, and then re-opens itself. The follower index is unavailable
+for reads and cannot replicate writes during this cycle.
+
+[discrete]
+[[ccr-remote-recovery]]
+=== Remote recovery process
+When you create a follower index, you cannot use it until it is fully
+initialized. The _remote recovery_ process builds a new copy of a shard on a
+follower node by copying data from the primary shard in the leader cluster.
+
+{es} uses this remote recovery process to bootstrap a follower index using the
+data from the leader index. This process provides the follower with a copy of
+the current state of the leader index, even if a complete history of changes
+is not available on the leader due to Lucene segment merging.
+
+Remote recovery is a network intensive process that transfers all of the Lucene
+segment files from the leader cluster to the follower cluster. The follower
+requests that a recovery session be initiated on the primary shard in the
+leader cluster. The follower then requests file chunks concurrently from the
+leader. By default, the process concurrently requests five 1MB file
+chunks. This default behavior is designed to support leader and follower
+clusters with high network latency between them.
+
+TIP: You can modify dynamic <<ccr-recovery-settings,remote recovery settings>>
+to rate-limit the transmitted data and manage the resources consumed by remote
+recoveries.
+
+Use the <<cat-recovery,recovery API>> on the cluster containing the follower
+index to obtain information about an in-progress remote recovery. Because {es}
+implements remote recoveries using the
+<<snapshot-restore,snapshot and restore>> infrastructure, running remote
+recoveries are labelled as type `snapshot` in the recovery API.
+
+[discrete]
+[[ccr-leader-requirements]]
+=== Requirements for leader indices
```

Comment:
Agreed -- it's good to provide a link for how users can configure this setting. I moved the history retention settings to the [Index modules](https://elasticsearch_60555.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/index-modules.html#_static_index_settings) settings page, and then included them in the History retention page to single source the content. 

I think that this addition (with a link on the `index.soft_deletes.retention_lease.period` setting) provides more context:

>The `index.soft_deletes.retention_lease.period` setting defines the maximum time to retain a shard history retention lease before it is considered expired. This setting determines how long the cluster containing your leader index can be offline, which is 12 hours by default. If a shard copy recovers after its retention lease expires, then {es} will fall back to copying the entire index, because it can no longer replay the missing history.